cmd/gctcli/commands.go

```diff
@@ -2027,32 +2026,207 @@ var withdrawCryptocurrencyFundsCommand = cli.Command{
 			Name:  "cryptocurrency",
 			Usage: "the cryptocurrency to withdraw funds from",
 		},
+		cli.StringFlag{
+			Name:  "address",
+			Usage: "address to withdraw to",
+		},
+		cli.StringFlag{
+			Name:  "addresstag",
+			Usage: "address tag/memo",
+		},
+		cli.Float64Flag{
+			Name:  "amount",
+			Usage: "amount of funds to withdraw",
+		},
+		cli.Float64Flag{
+			Name:  "fee",
+			Usage: "fee to submit with request",
+		},
+		cli.StringFlag{
+			Name:  "description",
+			Usage: "description to submit with request",
+		},
 	},
 }
 
-func withdrawCryptocurrencyFunds(_ *cli.Context) error {
-	return common.ErrNotYetImplemented
+// <exchange> <cryptocurrency> <address> <addresstag> <amount> <fee> <description>
+func withdrawCryptocurrencyFunds(c *cli.Context) error {
+	if c.NArg() == 0 && c.NumFlags() == 0 {
+		cli.ShowCommandHelp(c, "withdrawcryptofunds")
+		return nil
+	}
+
+	var exchange, cur, address, addresstag, description string
+	var amount, fee float64
+
+	if !c.IsSet("exchange") {
+		if c.Args().Get(0) != "" {
+			exchange = c.Args().Get(0)
+		}
+	}
+
+	if !c.IsSet("cryptocurrency") {
+		if c.Args().Get(1) != "" {
+			cur = c.Args().Get(1)
+		}
+	}
+
+	if !c.IsSet("address") {
+		if c.Args().Get(2) != "" {
+			address = c.Args().Get(2)
+		}
+	}
+
+	if !c.IsSet("addresstag") {
+		if c.Args().Get(2) != "" {
+			addresstag = c.Args().Get(3)
+		}
+	}
+
+	if !c.IsSet("amount") {
+		if c.Args().Get(3) != "" {
+			amountStr, err := strconv.ParseFloat(c.Args().Get(4), 64)
+			if err == nil {
+				amount = amountStr
+			}
+		}
+	}
+
+	if !c.IsSet("fee") {
+		if c.Args().Get(3) != "" {
+			feeStr, err := strconv.ParseFloat(c.Args().Get(5), 64)
+			if err == nil {
+				fee = feeStr
+			}
+		}
+	}
+
+	if !c.IsSet("description") {
+		if c.Args().Get(2) != "" {
+			description = c.Args().Get(6)
```

Comment:
they are in correct good pickup

I am also on the fence about usage of unnamed params here (e.g just passing in order "address" "tag" "amount" "fee" instead of --address because i feel it could be error prone

Reply:
On these big arg ones it's very risky, but I think it's still nice to offer both (I personally always use keys). On the server side, our withdraw validation should pick up any issues in case the user bodges them and I see our validation checks only getting better as time goes

Reply:
Didn't mean to ignore the replies here back working on this today :tada: 

Yeah the main concern for me was the address being out of place that can cause issues as a lot of exchanges will send even if its not valid

But as @thrasher- pointed out i have improved and will continue to improve client/gct side validation to address these and make it as error-prone free as possible